SPECS/cockpit/cockpit.spec

```diff
@@ -0,0 +1,86 @@
+Summary:        A package to provide cockpit for mariner
+Name:           cockpit
+Version:        248
+Release:        1%{?dist}
+License:        LGPLv2+ AND MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/cockpit-project/cockpit
+Source0:        https://github.com/cockpit-project/%{name}/releases/download/%{version}/%{name}-%{version}.tar.xz
+BuildRequires:  autoconf
+BuildRequires:  automake
+BuildRequires:  e2fsprogs-devel
+BuildRequires:  gcc
+BuildRequires:  gettext >= 0.19.7
+BuildRequires:  glib-devel
+BuildRequires:  gnutls-devel >= 3.4.3
+BuildRequires:  json-glib-devel
+BuildRequires:  krb5-devel >= 1.11
+BuildRequires:  libssh-devel >= 0.8.5
+BuildRequires:  make
+BuildRequires:  openssl-devel
+BuildRequires:  pam-devel
+BuildRequires:  polkit-devel
+BuildRequires:  sed
+BuildRequires:  systemd-devel >= 235
+BuildRequires:  which
+BuildRequires:  zlib-devel
+
+%description
+cockpit for mariner
+
+%prep
+%setup -q
+mkdir -p %{buildroot}%{_sysconfdir}/pam.d
+
+%build
+./configure --sysconfdir=%{_sysconfdir} --prefix=%{_prefix} --disable-pcp --disable-doc
+make %{?_smp_mflags}
+
+%install
+make install DESTDIR=%{buildroot}
+cat > %{buildroot}%{_sysconfdir}/pam.d/cockpit << EOF
+    #%PAM-1.0
+    # this MUST be first in the "auth" stack as it sets PAM_USER
+    # user_unknown is definitive, so die instead of ignore to avoid subsequent modules mess up the error code
+    -auth      [success=done new_authtok_reqd=done user_unknown=die default=ignore]   pam_cockpit_cert.so
+    auth       substack     system-auth
+    auth       optional     pam_ssh_add.so
+    account    required     pam_nologin.so
+    account    include      system-account
+    password   include      system-password
+    session    required     pam_loginuid.so
+    session    optional     pam_keyinit.so force revoke
+    session    optional     pam_ssh_add.so
+    session    include      system-session
+EOF
+chmod -R go+rx %{buildroot}%{_datadir}/cockpit
+chmod o+rx %{buildroot}%{_sysconfdir}/cockpit
+
+%check
+make check
```

Comment:
I noticed today that the RUN_CHECK=y flag wasn't actually causing the build system to run the check section here. I'm not sure why, as it does run the check sections on the other specs in this PR.

Reply:
Newest push should fix all issues with the %check section.